boards/xtensa/esp32/esp32-devkitc/src/esp32_mcp2515.c

```diff
@@ -0,0 +1,250 @@
+/****************************************************************************
+ * boards/xtensa/esp32/esp32devkitc/src/esp32_mcp2515.c
```

Comment:
Why is this not part of the common board logic?
I checked stm32 and it also there only stm32f3-minimum and not in the common folder.  Is there any particularities?  I can't see any from my side.

Reply:
you are right, it could become a common board logic, I will modify it!